llmfoundry/models/hf/hf_causal_lm.py

```diff
@@ -337,6 +338,17 @@ def build_inner_model(
         if dist.get_local_rank() == 0:
             os.remove(signal_file_path)
 
+        # Use the pretrained generation config for the model if it exists.
+        try:
```

Comment:
hmm, might need to do this on local rank 0 to prevent hangs the same way we do this with the model config

Reply:
done